packages/integrations/cloudflare/src/index.ts

```diff
@@ -229,10 +249,17 @@ export default function createIntegration(args?: Options): AstroIntegration {
 				_config = config;
 				finalBuildOutput = buildOutput;
 
+				let customWorkerEntryPoint: URL | undefined;
+				if (args?.workerEntryPoint?.path && typeof args.workerEntryPoint.path === 'string') {
+					customWorkerEntryPoint = new URL(args.workerEntryPoint.path, config.root);
+				}
+
 				setAdapter({
 					name: '@astrojs/cloudflare',
-					serverEntrypoint: '@astrojs/cloudflare/entrypoints/server.js',
-					exports: ['default'],
+					serverEntrypoint: customWorkerEntryPoint
+						? customWorkerEntryPoint
+						: '@astrojs/cloudflare/entrypoints/server.js',
+					exports: args?.workerEntryPoint?.exports ? args.workerEntryPoint.exports : ['default'],
```

Comment:
Maybe I missed a comment but didn't we say `'default'` should always be included even when `args.workerEntryPoint.exports` is provided?

```suggestion
					exports: args?.workerEntryPoint?.exports ? [...new Set(['default', ...args.workerEntryPoint.exports])] : ['default'],
```

Reply:
No, the docs really show that if you provide the value, you must provide `default`.